docs/root/intro/arch_overview/advanced/matching/matching_api.rst

```diff
@@ -179,8 +179,8 @@ HTTP Routing Integration
 ########################
 
 The matching API can be used with HTTP routing, by specifying a match tree as part of the virtual host
-and specifying a Route as the resulting action. See examples in the above sections for how the match
-tree can be configured.
+and specifying a Route or RouteListAction as the resulting action. See :ref:`the examples
```

Comment:
nit: please ref link Route and RouteListAction. Though now that I'm looking at this perhaps just s/Route/RouteList to make it consistent? Sorry for the churn.